es/LEEME.md

```diff
@@ -0,0 +1,222 @@
+
+* [3.0: Entendiendo Su Configuración Bitcoin](03_0_Entendiendo_Su_Configuracion_Bitcoin.md)
+  * [3.1: Verificando Su Configuración Bitcoin](03_1_Verificando_Su_Configuracion_Bitcoin.md)
+  * [3.2: Conociendo Su Configuración Bitcoin](03_2_Conociendo_Su_Configuracion_Bitcoin.md)
+  * [3.3: Configurando Su Billetera](03_3_Configurando_Su_Billetera.md)
+    * [Interludio: Usando Variables de la Interfaz de Linea de Comandos](03_3__Interludio_Usando_Variables_de_la_Interfaz_de_Linea_de_Comandos.md)
+  * [3.4: Recibiendo una Transacción](03_4_Recibiendo_una_Transaccion.md)
+  * [3.5: Entendiendo el Descriptor](03_5_Entendiendo_el_Descriptor.md)
+* [4.0: Enviando Transacciones Bitcoin](04_0_Enviando_Transacciones_Bitcoin.md)
+  * [4.1: Enviando Monedas de la Forma Fácil](04_1_Enviando_Monedas_de_la_Forma_Facil.md)
+  * [4.2: Creando una Transacción Cruda](04_2_Creando_una_Transaccion_Cruda.md)
+     * [Interludio: Usando JQ](04_2__Interludio_Usando_JQ.md)
+  * [4.3: Creando una Transacción Cruda con Argumentos Ingresados con Nombre](04_3_Creando_una_Transaccion_Cruda_con_Argumentos_Ingresados_con_Nombre.md)
+  * [4.4: Enviando Monedas con una Transacción Cruda](04_4_Enviando_Monedas_con_una_Transaccion_Cruda.md)
+     * [Interludio: Usando Curl](04_4__Interludio_Usando_Curl.md)
+  * [4.5: Enviando Monedas con Transacciones Crudas Automatizadas](04_5_Enviando_Monedas_con_Transacciones_Crudas_Automatizadas.md)
+  * [4.6: Creando una Transacción Segwit](04_6_Creando_una_Transaccion_Segwit.md)
+* [5.0: Controlando Transacciones Bitcoin](05_0_Controlando_Transacciones_Bitcoin.md)
+  * [5.1 Vigilando por Transacciones Estancadas](05_1_Vigilando_por_Transacciones_Estancadas.md)
+  * [5.2: Reenviando una Transacción con RBF](05_2_Reenviando_a_Transaccion_con_RBF.md)
+  * [5.3: Financiando una Transacción con CPFP](05_3_Financiando_una_Transaccion_con_CPFP.md)
+* [6.0: Expandiendo las Transacciones Bitcoin con Multifirmas](06_0_Expandiendo_las_Transacciones_Bitcoin_con_Multifirmas.md)
+  * [6.1: Enviando una Transacción a una Dirección Multifirma](06_1_Enviando_una_Transaccion_a_una_Direccion_Multifirma.md)
+  * [6.2: Gastando una Transacción con una Dirección Multifirma](06_2_Gastando_una_Transaccion_con_una_Direccion_Multifirma.md)
+  * [6.3: Enviando y Gastando una Multifirma Automatizada](06_3_Enviando_una_Multifirma_Automatizada.md)
+* [7.0: Expandiendo las Transacciones Bitcoin con PSBTs](07_0_Expandiendo_las_Transacciones_Bitcoin_con_PSBTs.md)
+  * [7.1: Creando una Transacción Bitcoin Parcialmente Firmada](07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md)
+  * [7.2: Usando una Transacción Bitcoin Parcialmente Firmada](07_2_Usando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md)
+  * [7.3: Integrando con Hardware Wallets](07_3_Integrando_con_Hardware_Wallets.md)
+* [8.0: Expandiendo Transacciones Bitcoin en Otros Sentidos](08_0_Expandiendo_Transacciones_Bitcoin_en_Otros_Sentidos.md)  
```

Comment:
08_0_Expandiendo_Bitcoin_Transacciones_Otros.md

Reply:
How the link name should be?

Reply:
I will kept the link name, because it's a literal translation from the english one.